tests/cases/unittests/versionCache.ts

```diff
@@ -159,50 +180,70 @@ and grew 1cm per day`;
     });
 
     describe('VersionCache stress test', () => {
-        const iterationCount = 20;
-        //const interationCount = 20000; // uncomment for testing
-
-        // Use scanner.ts, decent size, does not change frequentlly
-        let testFileName = "src/compiler/scanner.ts";
-        let testContent = Harness.IO.readFile(testFileName);
-        let totalChars = testContent.length;
-        assert.isTrue(totalChars > 0, "Failed to read test file.");
-
-        let {lines, lineMap} = server.LineIndex.linesFromText(testContent);
-        assert.isTrue(lines.length > 0, "Failed to initialize test text. Expected text to have at least one line");
-
-        let lineIndex = new server.LineIndex();
-        lineIndex.load(lines);
-
         let rsa: number[] = [];
         let la: number[] = [];
         let las: number[] = [];
         let elas: number[] = [];
         let ersa: number[] = [];
         let ela: number[] = [];
-        let etotalChars = totalChars;
+        const iterationCount = 20;
+        let lines: string[];
+        let lineMap: number[];
+        let lineIndex: server.LineIndex;
+        let testContent: string;
 
-        for (let j = 0; j < 100000; j++) {
-            rsa[j] = Math.floor(Math.random() * totalChars);
-            la[j] = Math.floor(Math.random() * (totalChars - rsa[j]));
-            if (la[j] > 4) {
-                las[j] = 4;
-            }
-            else {
-                las[j] = la[j];
-            }
-            if (j < 4000) {
-                ersa[j] = Math.floor(Math.random() * etotalChars);
-                ela[j] = Math.floor(Math.random() * (etotalChars - ersa[j]));
-                if (ela[j] > 4) {
-                    elas[j] = 4;
+        before(() => {
+            //const interationCount = 20000; // uncomment for testing
+
+            // Use scanner.ts, decent size, does not change frequentlly
+            let testFileName = "src/compiler/scanner.ts";
+            testContent = Harness.IO.readFile(testFileName);
+            let totalChars = testContent.length;
+            assert.isTrue(totalChars > 0, "Failed to read test file.");
+
+            ({lines, lineMap} = server.LineIndex.linesFromText(testContent));
+            assert.isTrue(lines.length > 0, "Failed to initialize test text. Expected text to have at least one line");
+
+            lineIndex = new server.LineIndex();
+            lineIndex.load(lines);
+
+            let etotalChars = totalChars;

```

Comment:
have no idea. This PR reorganizes the code preserving the test logic in the way than data allocated in `before` step can be cleaned up in `after` step